f2/src/forms/MidFeedbackForm.js

```diff
@@ -148,7 +148,7 @@ export const MidFeedbackForm = props => {
                   <Field name="problemDescription">
                     {props => (
                       <FormControl>
-                        <FormLabel>
+                        <FormLabel htmlFor="problemDescription">
```

Comment:
This is fixed by #1496 We haven't merged it yet

Reply:
We'll need to be careful when merging into master. We have an other PR for this file.

Reply:
will remove it from this pr shortly

Reply:
@amazingphilippe  this is now removed from this PR